adhocracy-plus/assets/scss/components/_a4-comments.scss

```diff
@@ -124,18 +124,34 @@
 
 .a4-comments__action-bar__btn {
     margin-left: 0.2 * $spacer !important;
+    text-decoration: underline;
 
     @extend .btn--small;
 
     &:first-child {
         margin-left: 0 !important;
     }
+
+    &:hover {
+        text-decoration: none;
+    }
 }
 
 .a4-comments__border--highlighted {
     border-left: 2px solid $brand-primary;
 }
 
+.btn.btn--none.a4-comments__read-btn {
+    text-decoration: underline;
+    margin-bottom: 0.5 * $spacer;
```

Comment:
the issue asks for padding on the side, but there's only a margin at the bottom now, is this correct?

Reply:
that was on purpose, adding padding would have meant a4 refactor so I just made the buttons the act same as other comment buttons, it didn't seem vital enough with the time limit but not sure if that's changed?